pkg/util/env/env.go

```diff
@@ -134,3 +135,27 @@ func GetAntreaNamespace() string {
 func GetAllowNoEncapWithoutAntreaProxy() bool {
 	return getBoolEnvVar(allowNoEncapWithoutAntreaProxyEnvKey, false)
 }
+
+func GetKubeServiceHost() string {
+	kubeServiceHost := os.Getenv(k8s.KubeServiceHostEnvKey)
+	if kubeServiceHost == "" {
+		klog.Warningf("Environment variable %s not found", k8s.KubeServiceHostEnvKey)
+	}
+	return kubeServiceHost
+}
+
+func GetKubeServicePort() int32 {
+	kubeServicePortStr := os.Getenv(k8s.KubeServicePortEnvKey)
+
+	if kubeServicePortStr == "" {
+		klog.Warningf("Environment variable %s not found", k8s.KubeServicePortEnvKey)
+	} else {
+		parsedValue, err := strconv.ParseInt(kubeServicePortStr, 10, 16)
```

Comment:
```suggestion
		parsedValue, err := strconv.ParseInt(kubeServicePortStr, 10, 32)
```

Reply:
Why is 32 used here? The port is a 16-bit number.